src/maintainer/adding_pkgs.rst

```diff
@@ -110,6 +110,36 @@ Post staging process
 * If you want to make a change to the recipe, send a :term:`PR` to the git repository from a fork. Branches of the main repository are used for maintaining different versions only.
 
 
+.. _feedstock-repository-structure:
+
+Feedstock repository structure
+------------------------------
+
+Once the PR having the recipe for a package is merged in the ``staged-recipes`` repository, a new repository is created automatically called ``<package-name>-feedstock``.
```

Comment:
I feel like "having" isn't the best word here. Maybe "containing" or just "for", though then we could have two instances of the word "for" pretty close together. Up to you.